src/python/foglamp/sending_process.py

```diff
@@ -658,12 +653,30 @@ def _last_object_id_update(self, new_last_object_id, stream_id):
         try:
             _logger.debug("Last position, sent |{0}| ".format(str(new_last_object_id)))
 
-            sql_cmd = "UPDATE foglamp.streams SET last_object={0}, ts=now()  WHERE id={1}" \
-                .format(new_last_object_id, stream_id)
+            # TODO : to be changed using PayloadBuilder after FOGL-616
+            # TODO : verify the support for .SET(rs='now()')
+            # payload = payload_builder.PayloadBuilder() \
+            #     .WHERE(['column', '=', stream_id]) \
+            #     .SET(last_object=new_last_object_id) \
+            #     .SET(rs='now()') \
```

Comment:
rs? You mean ts? As per my understanding ts column is db specific, not to be used in code. ts should be updated by db itself on insert/update.

Reply:
you are right, it is 'ts' - changed. but I don't think the DB is updating that field autonomously, am I wrong ?

Reply:
verified : 
- the 'ts' field is not updated automatically
- the current code that doesn't use the PayloadBuilder, is :
            values = dict()
            values['last_object'] = new_last_object_id
            values['ts'] = 'now()'

and it works fine.